packages/astro/src/vite-plugin-markdown/index.ts

```diff
@@ -147,6 +152,8 @@ ${setup}`.trim();
 					site: config.site ? new URL(config.base, config.site).toString() : undefined,
 					sourcefile: id,
 					sourcemap: 'inline',
+					// TODO: baseline flag
```

Comment:
just double checking, this a future PR todo correct?

Reply:
Yeah, this was copy + pasted from the Astro plugin, I think a note from @matthewp